kernel/core/commandfunctions.hpp

```diff
@@ -3142,6 +3142,33 @@ static CommandReturnValues cmd_clear(string& sCmd)
 
         return cmd_delete(sCommand);
     }
+    else if (cmdParser.hasParam("var"))

```

Comment:
```suggestion
    else if (cmdParser.hasParam("var") && NumeReKernel::getInstance()->getDebugger().getStackSize() == 0)
```
You want to ensure that this is only allowed within the terminal out of any procedures or flow control statements.